Rakefile

```diff
@@ -109,4 +109,35 @@ namespace :breathe do
       File.write("tmp/data/breathe/#{person.emails.first}.json", JSON.generate(person_data))
     end
   end
+
+  desc "Do a verbose dry run from files of BreatheHR data"
+  task :dry_run_data_dump_to_productive do
+    puts "Simulating syncing events from tmp/data/breathe/*.json"
```

Comment:
```suggestion
    puts "Doing a dry run!"
    puts
    puts "This task always does a dry run."
    puts "You must use the Breathe API to make real changes:"
    puts
    puts "$ bundle exec rake breathe:to_productive"
    puts
```

Reply:
Ah, I see, the above reply was left before I got to the end of the suggestions.